ktor-shared/ktor-openapi-schema/jvm/test-resources/petstore.json

```diff
@@ -0,0 +1,236 @@
+{
+  "openapi": "3.0.0",
```

Comment:
Btw, what is the max supported OpenAPI version? I see that [v3.2.0 has been published](https://spec.openapis.org/oas/v3.2.0.html) in September.